nginx_pm2_krakenjs/nginx/nginx.conf

```diff
@@ -37,6 +38,8 @@ http {
             proxy_read_timeout 200;
             proxy_http_version 1.1;
             proxy_set_header Connection "";
+            proxy_next_upstream http_500 http_502 http_503;
```

Comment:
`500` we should not retry because its already a server error. 
`502` and `503` 👍